### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by ad2ien**
#358 

- terraform provider
- helm charts

K8s, keycloak and Synapse are updated by env var. Tested on lagraulet env.

Also add some resource definitions

### CODE REVIEW

File: ansible/roles/keycloak/templates/keycloak.yml.j2
Comment (heroinedor) [inline ansible/roles/keycloak/templates/keycloak.yml.j2:1]: Limitations based on which measurement ?

File: ansible/roles/monitoring/templates/matrix-servicemonitor.yml
Comment (heroinedor) [inline ansible/roles/monitoring/templates/matrix-servicemonitor.yml:1]: Limitations based on which measurement ?

File: ansible/roles/synapse/templates/synapse-chart-config.j2
Comment (heroinedor) [inline ansible/roles/synapse/templates/synapse-chart-config.j2:1]: Limitations based on which measurement ?

File: ansible/roles/element-web/tasks/main.yml
Comment (heroinedor) [inline ansible/roles/element-web/tasks/main.yml:1]: Limitations based on which measurement ?

**Review by heroinedor — APPROVED** [review]

**Comment by ad2ien** [comment]
Resources based on what I could find on the web and tested on lagraulet env with pod monitoring dashboard on grafana

**Pull request merged**